### PR TITLE
Support Ruby 3.2.2

### PR DESCRIPTION
**Pull request by jmarrec**

### CODE REVIEW

**Comment by wenyikuang** [comment]
Disabled.

File: Gemfile [hunk 1]
```diff
@@ -3,32 +3,107 @@
...
-  gem 'msgpack', '1.4.2'
+LOCAL_DEV = false
+MINIMAL_GEMS = false   # Keep only one non-native gem, and one native
+FINAL_PACKAGE = !ENV['FINAL_PACKAGE'].nil?
+
+if !MINIMAL_GEMS
+  # Bug in addressable to 2.8.1 and patched version has an issue https://github.com/NREL/OpenStudio/issues/4870
+  gem 'addressable', '= 2.8.1'
+  gem 'openstudio-standards', '= 0.5.0'
+  # gem 'json_schemer', '= 2.0.0' # Disabled, see #72 and https://github.com/NREL/OpenStudio/issues/4969#issuecomment-1943418472
```
Comment (shorowit) [inline Gemfile:22]: @jmarrec Any chance the `json_schemer` gem happens to work w/ ruby 3.2.2?

Comment (jmarrec) [reply Gemfile:22]: Definitely a certainty that I disabled it on purpose unfortunately on this PR.

It's due to unf_ext, see #72

Comment (jmarrec) [reply Gemfile:22]: I mean... let me rephrase.

We can include it, but if we do, then you can only build openstudio SDK in Release, and it'll fail in debug.

@kbenne FYI

That means we would need a Release and a Debug package of openstudio3-gems for windows... That's not an awfully complicated thing to do I guess, but it has been very very low on my long todo list to get this working.

I'm assuming this is important to you. How critical @shorowit ?

Comment (kbenne) [reply Gemfile:22]: I don't think we should hold up merging this and moving forward with an OpenStudio release candidate for a new gem.

**Review by kbenne — APPROVED** [review]

**Comment by jmarrec** [comment]
@wenyikuang or @kbenne has one of you enough juice to go to the Repo Settings and remove the Jenkins status checks from the Required status to merge a PR?

File: openstudio-gems.gemspec [hunk 2]
```diff
@@ -18,14 +18,14 @@ Gem::Specification.new do |spec|
...
-  spec.add_dependency 'openstudio_measure_tester', '~> 0.3.2'
+  spec.add_dependency 'openstudio-extension', '0.8.0'
+  spec.add_dependency 'openstudio-workflow', '2.4.0'
+  spec.add_dependency 'openstudio_measure_tester', '~> 0.4.0'
 
   spec.add_dependency 'parallel', '1.19.1'
 
   # 20200324 TJC json native is enabled for all platforms and is included when building ruby
-  #spec.add_dependency 'json_pure', '2.2'
+  spec.add_dependency 'json_pure', '2.2'
```
Comment (shorowit) [inline openstudio-gems.gemspec:28]: @jmarrec Was this intentional? I think everyone would prefer to use the native json, not the json_pure gem. It was [previously disabled](https://github.com/NREL/openstudio-gems/commit/a4da055ad9c1361634b96f9c3754fa3f28229e42).

Comment (jmarrec) [reply openstudio-gems.gemspec:28]: I probably messed up. @kbenne if you can revert if needed that'd be great
